turbinia/workers/__init__.py

```diff
@@ -1197,23 +1186,3 @@ def run(self, evidence, result):
         TurbiniaTaskResult object.
     """
     raise NotImplementedError
-
-  def update_task_status(self, task, status=None):
```

Comment:
I think we will want to update the task status from the task in order to get mid-run updates, but we can always add this or similar back in when we actually utilize that.